frankenphp.c

```diff
@@ -105,13 +106,23 @@ static uintptr_t frankenphp_clean_server_context() {
 
 static void frankenphp_request_reset() {
   zend_try {
+    frankenphp_server_context *ctx = SG(server_context);
+
     int i;
+    if (ctx->worker_ready) {
+      // Restore worker script super globals
+      for (i = 0; i < NUM_TRACK_VARS; i++) {
+        zval_ptr_dtor(&PG(http_globals)[i]);
+      }
+      memcpy(&PG(http_globals), &ctx->worker_http_globals,
```

Comment:
This doesn't work because `php_hash_environment()` clears `PG(http_globals)`): https://github.com/php/php-src/blob/62cdaac2806ba0a9f3443508ba60a6f552ef6bce/main/php_variables.c#L787